src/styles/calendar.scss

```diff
@@ -0,0 +1,22 @@
+.calendar {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  box-sizing: border-box;
```

Comment:
move to `*` selector

Reply:
fixed

Reply:
u can delete it here, because u have set it to everything